src/input.rs

```diff
@@ -101,6 +108,13 @@ impl Input {
         });
         self.0.mouse_pos = pos;
     }
+
+    pub fn mouse_wheel(&mut self, delta: MouseScrollDelta) {
+        self.1.mouse_wheel += match delta {
```

Comment:
I'm not sure is it possible to make two different mouse wheel moves during one frame...